src/client/components/rich-text/rich-text.vue

```diff
@@ -1,9 +1,66 @@
 <template>
-  <div v-html="text"></div>
+  <div class="rich-text" v-html="text"></div>
 </template>
 
 <script>
 export default {
   props: ['text']
 }
 </script>
+
+<style>
+.rich-text {
+  width: 100%;
+  max-width: 600px;
```

Comment:
`600px` is being used for setting `max-width` multiple times now, so I think we should move it to a variable.